apps/Marathon/install

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+version=20220115
+
+#made according to the instructions from the developer, but was not originally created for the Pi
+#https://github.com/Aleph-One-Marathon/alephone/wiki/Linux%20Install%20Instructions
+
+install_packages libboost-all-dev libsdl2-dev \
+  libsdl2-image-dev libsdl2-net-dev libsdl2-ttf-dev \
+  libspeexdsp-dev libzzip-dev libavcodec-dev libavformat-dev \
+  libavutil-dev libswscale-dev libswresample-dev libpng-dev libcurl4-openssl-dev libminiupnpc-dev build-essential unzip || exit 2
+
+
+#creating a directory for game files and remove previous gamefiles
+rm -rf  ~/AlephOneData
+mkdir -p ~/AlephOneData
+
+#installation of the Marathon game engine (Aleph One)
+cd /tmp
+wget https://github.com/Aleph-One-Marathon/alephone/releases/download/release-${version}/AlephOne-${version}.tar.bz2 || exit 1
+
+status "Unzipping..."
+rm -f AlephOne-${version}
+tar xjvf AlephOne-${version}.tar.bz2 || error "Unable to decompress source code"
+rm -f AlephOne-*.tar.bz2
+status_green "Done"
+
+status "Compilation..."
+cd AlephOne-${version}
+
+# get the $DISTRIB_RELEASE by calling lsb_release
+# check if upstream-release is available
+if [ -f /etc/upstream-release/lsb-release ]; then
+  echo "This is a Ubuntu Derivative, checking the upstream-release version info"
+  DISTRIB_CODENAME=$(lsb_release -s -u -c)
+else
+  DISTRIB_CODENAME=$(lsb_release -s -c)
+fi
```

Comment:
Should these lines be replaced with the `get_codename` function?

Reply:
not sure I like the get_codename function... I worried that the apt output would pollute the variable with stdout
```
get_codename() { #get debian/ubuntu codename
  if ! command -v lsb_release &>/dev/null; then
    apt_update && sudo apt-get install -y lsb-release
  fi
  
  # check if upstream-release is available
  if [ -f /etc/upstream-release/lsb-release ]; then
    # this is a derivative, get the upstream release info
    lsb_release -suc
else
    lsb_release -sc
  fi
}
```

Reply:
I verified get_codename is a fine replacement, switching now